### PR TITLE
Fix impersonate

### PR DESCRIPTION
**Pull request by ignaciojonas**

### CODE REVIEW

**Comment by falvarisqueta** [comment]
Looks good to me except that CI failed

File: spec/integration/lib/auth0/api/api_authentication_spec.rb [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    @impersonator_user = client.create_user(impersonator_username, 'email' => impersonator_email,
+                                                                   'password' => password,
+                                                                   'email_verified' => true,
+                                                                   'connection' =>
+                                                                    Auth0::Api::AuthenticationEndpoints::UP_AUTH,
+                                                                   'app_metadata' => {})
+  end
+
+  after(:all) do
+    client = Auth0Client.new(v2_creds)
```
Comment (leosoubeste) [inline spec/integration/lib/auth0/api/api_authentication_spec.rb]: I think this is no longer needed as new HttpProxy is merged into master.

Comment (ignaciojonas) [inline spec/integration/lib/auth0/api/api_authentication_spec.rb]: Good catch, I'll fix this..

**Comment by leosoubeste** [comment]
Looks good to me.

**Pull request merged**